test/integration/server/add_test.go

```diff
@@ -220,7 +227,7 @@ var _ = Describe("AddApplication", func() {
 			}
 		})
 	})
-	Context("via auto merge", func() {
+	Context("[github] via auto merge", func() {
```

Comment:
I think we could rely on `Context("Github")` and `Context("Gitlab")` instead of adding them in `[]`

Reply:
Wrapped everything in `Context` blocks :+1: